spec/imsc-hrm.html

```diff
@@ -200,7 +200,19 @@
   </section>
 
   <section id='sotd'>
-   
+    <p>For this specification to exit the CR stage, the following independent factors of verifications MUST be demonstrated:</p>
+      <ul>
+        <li>at least two content producing implementations</li>
+        <li>at least one validating implementation</li></p>
+      </ul>
+      <p>The Working Group has identified the following <a href="https://www.w3.org/2021/Process-20211102/#at-risk">at
+      risk</a> feature: validation of documents that conform to the IMSC Image Profile. This feature is expected to be
```

Comment:
It'd be neat if we could mark the at-risk features up within the document, and have them automatically gathered here. I think that's a Respec feature request though!

Reply:
I think you may choose both, like having list of at-risk in SoTD for [CSP3](https://www.w3.org/TR/2022/WD-CSP3-20221201/#sotd), or noting them with [class=issue in old permissions spec](https://www.w3.org/TR/2021/WD-permissions-20211123/#dfn-at-risk).

Reply:
Opening a "remove at risk feature XXX" issue and adding a link to it in the appropriate part of the spec seems like a good way to track the at risk features, to me. We can add an "at risk" label to the issue(s) too, for easy filtering.

Reply:
Although I love the idea, I do not see how this can be implemented in a straightforward fashion given the nature of the at-risk feature.

Reply:
An issue could be put at the start of §9. Paint Images, say.

Reply:
I dislike the idea of a proliferation of notes. The one place I could see adding a note would be under the definition of _IMSC Document Instance_, which currently includes both Image and Text Profile documents.

Reply:
all right, will try something.

Reply:
8cdd961 looks good to me, validation failure aside. Not sure if we really need the at risk feature to be listed in the Conformance section, but I can see the reasons for putting it everywhere you have done @palemieux .